### ISSUE TITLE
Hi Awesome Code can you please tell me what do need to do to open specific folder as main root

### ISSUE THREAD

**Issue by blazingdevil44** [praharshjain/Vudit#6]
Hi thanks for sharing an awesome code can you please tell me a why i can open specific folder as main path instead of all whole external path e.g i have folder in pictures of my app i dont need everything else i just need to open my App folder as main path so it only shows its files and folders or documents of that specific folder when i try to leave my app folder by backpress it should leave fileviewer

Long story short i just want to open my own app folder with fileviewer

**Comment by praharshjain** [comment]
@blazingdevil44 you just need to call `updateFiles` with the file object created from the path to given folder. For example, 
`updateFiles(Environment.getExternalStorageDirectory());`
